### PR TITLE
Add puppy token contract

### PR DESCRIPTION
**Pull request by lohsuan**
## Description
import PuppyTokenContract and add contract reference to Service page.

## How to verify
run it

## Screenshot
image

### CODE REVIEW

File: src/components/Services.jsx [hunk 1]
```diff
@@ -21,6 +23,26 @@ const Services = () => (
...
           Offer you convenient, transparent and rapid services to donate safely around the world.
         </p>
+        <a
+          href={`https://ropsten.etherscan.io/address/${contractAddress}`}
+          target="_blank"
+          rel="noreferrer"
+          className="text-left my-2 text-white font-light md:w-9/12 w-11/12 text-base hover:text-blue-400"
+        >
+          <p>
+            Contract address: {shortenAddress(contractAddress)}
```
Comment (Xanonymous-GitHub) [inline src/components/Services.jsx]:
```suggestion
            Contract address: <b className="text-cyan-300">{shortenAddress(contractAddress)}</b>
```

Suggest adding colors to the contract address for emphasis.

File: src/components/Services.jsx [hunk 2]
```diff
@@ -21,6 +23,26 @@ const Services = () => (
...
+          </p>
+        </a>
+        <a
+          href={`https://ropsten.etherscan.io/address/${tokenAddress}`}
+          target="_blank"
+          rel="noreferrer"
+          className="text-left my-2 text-white font-light md:w-9/12 w-11/12 text-base hover:text-blue-400"
+        >
+          <p>
+            Token address: {shortenAddress(tokenAddress)}
```
Comment (Xanonymous-GitHub) [inline src/components/Services.jsx]: ditto

Comment (lohsuan) [reply src/components/Services.jsx]: resolve here [daface6](https://github.com/lohsuan/puppy_sponsor/pull/13/commits/daface6eb13f0406e6b1ac018a3b58ce09b2ae4f)

File: src/components/Services.jsx (hunk 1 above)
Comment (lohsuan) [reply src/components/Services.jsx]: resolve here [daface6](https://github.com/lohsuan/puppy_sponsor/pull/13/commits/daface6eb13f0406e6b1ac018a3b58ce09b2ae4f)

**Review by Xanonymous-GitHub — APPROVED** [review]
LGTM

**Pull request merged**